example/src/main/java/com/stripe/example/activity/CreateCardPaymentMethodActivity.kt

```diff
@@ -0,0 +1,158 @@
+package com.stripe.example.activity
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.snackbar.Snackbar
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.Stripe
+import com.stripe.android.model.Address
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.example.R
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.activity_card_payment_methods.*
+
+class CreateCardPaymentMethodActivity : AppCompatActivity() {
+    private val compositeDisposable = CompositeDisposable()
+
+    private lateinit var adapter: PaymentMethodsAdapter
+    private lateinit var stripe: Stripe
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_card_payment_methods)
+
+        stripe = Stripe(
+            applicationContext,
+            PaymentConfiguration.getInstance(this).publishableKey
+        )
+
+        adapter = PaymentMethodsAdapter()
+        val paymentMethods: RecyclerView = findViewById(R.id.payment_methods)
```

Comment:
Started reading up on Kotlin view binding extensions when I saw the `btn_create_payment_method` line below. Now I'm curious... can you replace this line with:

`val paymentMethods: RecyclerView = payment_methods`

Or is there a reason you're preferring findViewById instead here?

If you can... could you also replace each occurrence of `findViewById` with the synthetic reference version in this file?

Reply:
Will do!